docs/validators/sigstore_cosign.md

```diff
@@ -136,6 +140,30 @@ kubectl create secret generic my-secret \
 In the above case, the secret name in Connaisseur configuration would be `secret_name: my-secret`.
 It is possible to provide one Kubernetes secret with a `config.json` for authentication to multiple private registries and referencing this in multiple validators.
 
+#### k8s_keychain
+
+Specify `k8s_keychain: true` in the validator configuration to pass the flag `--k8s-keychain` to `cosign` when performing image validation.
+This will allow `cosign` to pick up ambient registry credentials from the environment.
```

Comment:
minor proposal:
```suggestion
Specification of `auth.k8s_keychain: true` in the validator configuration passes the `--k8s-keychain` to `cosign` when perfoming image validation.
Thus, [k8schain](https://pkg.go.dev/github.com/google/go-containerregistry/pkg/authn/k8schain) is used by `cosign` to pick up ambient registry credentials from the environment and for example use workload identities in case of common cloud providers.
```
I was unable to find any better documentation of what it is and how it works. Would you be aware of anything?

Reply:
I used your suggested changes here.

Unfortunately, the flag doesn't seem to be documented outside the help message displayed by `cosign`, which says (from https://github.com/sigstore/cosign/blob/3ccf3c6a869eaed6329c4e717bfbc7738f0610a1/cmd/cosign/cli/options/registry.go#L49 ): 

`whether to use the kubernetes keychain instead of the default keychain (supports workload identity).`

I think linking to the k8schain documentation, along with the link I added to the cosign pull request implementing the flag, is sufficient for now.

Reply:
agreed